client/src/constants/messages.js

```diff
@@ -21,6 +20,10 @@ export const TICKETS =
   'This page allows you to submit a help request via an RT Ticket.';
 export const UI =
   'This hidden page allows developers to review UI components in isolation.';
+
+/**
+ * Error Message Texts
+ */
 export const GOOGLE_DRIVE_SETUP_ERROR = (
   <span>
```

Comment:
__Idea__
The `<span>` and `</span>` can probably be a [React fragment](https://reactjs.org/docs/fragments.html) (`<>` and `</>`) to avoid extra markup, but I don't have bandwidth to check whether the span is necessary, nor test this change, so I make no edit.